include/libvfio-user.h

```diff
@@ -137,18 +137,21 @@ int
 vfu_get_poll_fd(vfu_ctx_t *vfu_ctx);
 
 /**
- * Polls the vfu_ctx and processes the command recieved from client.
+ * Polls the vfu_ctx and processes the command received from client.
  * - Blocking vfu_ctx:
  *   Blocks until new request is received from client and continues processing
  *   the requests. Exits only in case of error or if the client disconnects.
  * - Non-blocking vfu_ctx(LIBVFIO_USER_FLAG_ATTACH_NB):
  *   Processes one request from client if it's available, otherwise it
- *   immediatelly returns and the caller is responsible for periodically
+ *   immediately returns and the caller is responsible for periodically
  *   calling again.
  *
  * @vfu_ctx: The libvfio-user context to poll
  *
- * @returns 0 on success, -1 on error. Sets errno.
+ * @returns 0 on success, -1 on error, with errno set as follows:
+ *
+ * EAGAIN/EWOULDBLOCK: no more commands to process
```

Comment:
I didn't look at the code very closing but it seems to me that there can be other types of errno, e.g. `EINVAL`. Are we sure these are the only two errnos that can be set?

Reply:
done